lib/Elastica/Query.php

```diff
@@ -23,16 +23,16 @@
 class Query extends Param
 {
     /**
-     * Suggest query or not.
+     * If the current query has a suggest in it.
      *
-     * @var int Suggest
+     * @var bool Suggest
      */
-    protected $_suggest = 0;
+    protected $hasSuggest = false;
```

Comment:
Is it time to get rid of the protected `_` convention everywhere? The simple answer is yes, but how do we migrate there? Do we just adjust the new code to be different or all at once?

Reply:
Yes, I'd move all of those to proper camelCase and get rid of the `_` prefix for protected properties in a next PR

Reply:
I hear breaking change? Should we try to do it before 7.0 release?

Reply:
Done, changelog updated too